include/save.h

```diff
@@ -17,6 +17,8 @@
 #define SECTOR_ID_SAVEBLOCK2          0
 #define SECTOR_ID_SAVEBLOCK1_START    1
 #define SECTOR_ID_SAVEBLOCK1_END      4
+#define SECTOR_ID_SAVEBLOCK2_START    14
+#define SECTOR_ID_SAVEBLOCK2_END      27
```

Comment:
I think this has slipped in from a backup-save removal commit?

Reply:
No, this is me trying to guess what sectors these end in for use in `CheckSaveBlock2Size`.

Reply:
Ah! I think `CheckSaveBlock2Size` is fine, there's just a single sector dedicated to `SaveBlock2` (it's sector 0) :)